src/test/java/org/gridsuite/directory/server/DirectoryTest.java

```diff
@@ -899,6 +899,13 @@ public void testGetElement() {
                 toElementAttributes(scriptAttributes.getElementUuid(), "newScript", FILTER, null, "user1", null, scriptAttributes.getCreationDate())
             ))
         );
+
+        List<ElementAttributes> result = getElements(List.of(FILTER_UUID, UUID.randomUUID()), "user1", false, List.of(FILTER), 200);
```

Comment:
add a directory to the list to check that it is returned by this method

Reply:
ok done